backend/tracim_backend/tests/functional/test_webdav.py

```diff
@@ -223,3 +275,276 @@ def test_functional__webdav_access_to_workspace__workspace_not_exist(self) -> No
+        api = ContentApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+        )
+        with dbsession.no_autoflush:
+            file = api.create(
+                content_type_list.File.slug,
+                workspace,
+                None,
+                filename='file.txt',
+                do_save=False,
+                do_notify=False,
+            )
+            api.update_file_data(
+                file,
+                '/?\#*.txt',
+                'text/plain',
+                b'test_content'
+            )
+            api.save(file)
+        transaction.commit()
+
+        self.testapp.authorization = (
+            'Basic',
+            (
+                'admin@admin.admin',
+                'admin@admin.admin'
+            )
+        )
+        # /?\#* -> /⧸ʔ⧹#∗
+        # urlencoded version
+        self.testapp.get('/%E2%A7%B8%CA%94%E2%A7%B9%23%E2%88%97', status=200)
+        self.testapp.get('/%E2%A7%B8%CA%94%E2%A7%B9%23%E2%88%97/%E2%A7%B8%CA%94%E2%A7%B9%23%E2%88%97.txt', status=200)
+
+    def test_functional__webdav_access_to_content__err__file_not_exist(self) -> None:
+        dbsession = get_tm_session(self.session_factory, transaction.manager)
+        admin = dbsession.query(User) \
+            .filter(User.email == 'admin@admin.admin') \
+            .one()
+        workspace_api = WorkspaceApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+            show_deleted=True,
+        )
+        workspace = workspace_api.create_workspace('workspace1', save_now=True)  # nopep8
+        api = ContentApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+        )
+        transaction.commit()
+
+        self.testapp.authorization = (
+            'Basic',
+            (
+                'admin@admin.admin',
+                'admin@admin.admin'
+            )
+        )
+        self.testapp.get('/workspace1', status=200)
+        self.testapp.get('/workspace1/file.txt', status=404)
+
+    def test_functional__webdav_access_to_subdir_content__ok__nominal_case(self) -> None:
+        dbsession = get_tm_session(self.session_factory, transaction.manager)
+        admin = dbsession.query(User) \
+            .filter(User.email == 'admin@admin.admin') \
+            .one()
+        workspace_api = WorkspaceApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+            show_deleted=True,
+        )
+        workspace = workspace_api.create_workspace('workspace1', save_now=True)  # nopep8
+        api = ContentApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+        )
+        folder = api.create(
+            content_type_list.Folder.slug,
+            workspace,
+            None,
+            'folder',
```

Comment:
use realistic names. folder is not a name a user will give to a folder. For example set the folder name to "examples". Giving folder may let the reader think you create a content with a type `folder` or something similar

Reply:
you're right, i just change it.